src/store/rescan.js

```diff
@@ -30,6 +34,7 @@ export default {
         .then(result => {
           result.running = true;
           commit("setRescan", result);
+          commit("setLastClick", new Date());
```

Comment:
Can't we do this before starting the HTTP request? The change in the GUI would be more immediate I think.